src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue22972.cs

```diff
@@ -0,0 +1,22 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue22972 : _IssuesUITest
+	{
+		public Issue22972(TestDevice device) : base(device)
+		{
+		}
+
+		public override string Issue => "Win platform WebView cannot be release after its parent window get close";
+
+		[Test]
+		[Category(UITestCategories.WebView)]
+		public void WebViewDoesNotLeak()
+		{
+			App.WaitForElement("Success", timeout: TimeSpan.FromSeconds(10));
```

Comment:
If this fails here, do you see the full message on the test failure in AzDO?
```csharp
"Failed to cleanup: " + string.Join(", ", stillAlive)
```

Reply:
We get a screenshot of every test that fails. I could probably pull the failure text message though.